s3transfer/download.py

```diff
@@ -191,8 +191,8 @@ def _get_temp_fileobj(self):
         return f
 
     def _get_fileobj(self, filename):
-        f = DeferredOpenFile(filename, mode='wb')
-        f.OPEN_METHOD = self._osutil.open
+        f = DeferredOpenFile(
+            filename, mode='wb', open_method=self._osutil.open)
```

Comment:
Ahh, yeah I think that was because the class variable used to be called ``OPEN_METHOD`` and that threw me off a bit. I just updated it to be ``open_function``.